team.html

```diff
@@ -308,6 +308,33 @@ <h6 class="card-subtitle pt-1 pb-3" style="color: #EF4035">Faculty Adviser</h6>
                         </div>
                      </div>
                   </div>
+
+                  <div class="card my-2">
```

Comment:
Can you wrap this `card` div in another `<div class="card-deck">` like we did here: https://github.com/webdvt/webdvt/blob/89da4a324bda3810933a62f7cfab25911ee25a1e/team.html#L125. This is so that so the cards are not squished together. 

For the second `card-deck` div (the second row), replace Dr. Backs profile with Jared, and put Dr. Back's profile in the third `card-deck` div.